projects/uitdatabank/reference/entry.json

```diff
@@ -8301,7 +8301,86 @@
         ],
         "x-internal": true
       },
-      "parameters": []
+      "parameters": [],
+      "get": {
+        "summary": "ownership - get",
+        "operationId": "get-ownerships",
+        "responses": {
+          "200": {
+            "description": "A list of ownership requests",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "itemsPerPage": {
+                      "type": "integer",
+                      "example": 30,
+                      "description": "The amount of results that is being returned per page."
+                    },
+                    "totalItems": {
+                      "type": "integer",
+                      "example": 2345,
+                      "description": "Total amount of results for the given query parameters."
+                    },
+                    "member": {
+                      "type": "array",
+                      "description": "Search results (paginated).",
+                      "items": {
+                        "$ref": "../models/ownership.json"
+                      }
+                    }
+                  },
+                  "required": [
+                    "itemsPerPage",
+                    "totalItems",
+                    "member"
+                  ]
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/NotFound"
+          }
+        },
+        "tags": [
+          "Ownerships"
+        ],
+        "description": "Get all or one specific ownership requests of the current user",
```

Comment:
I don't think the current user is important. Also it doesn't return "all" ownerships, it are the ownerships matching the search query.

This makes me wonder if we need a dedicated endpoint `GET /ownerships/search` like `GET /ownerships/suggestions` to make the difference more clear with `GET /ownerships/{ownershipId}